src/feature-selector-form/feature-selector-form.module.css

```diff
@@ -144,6 +146,7 @@ button.clearButton:active {
 	max-width: 20rem;
 }
 
+.selectedFeatureRepositoriesTitle,
 .selectedFeatureKeywordTitle {
 	font-size: var( --font-body );
 	font-weight: 500;
```

Comment:
**Suggestion (non-blocking)**: Comparing to the mock-ups, the margin between these sections looks like it was a bit bigger than the mock-ups (which seem to be `1.25rem`). I think it's because these are `p` elements and browsers add an extra `1em` margin to the top.

I think if you add `margin-top: 0` here, it might tighten it up a little bit!

Reply:
Done! Fixed in [8c403ef](https://github.com/Automattic/bugomattic/pull/88/commits/8c403efcc0879e46f38643070cbdf79d6490470b).